frontend/server/src/DAO/Schools.php

```diff
@@ -106,6 +106,66 @@ public static function getRankByUsersAndProblemsWithAC(
         return $result;
     }
 
+    /**
+     * Returns the rank of schools based on the sum of the score of each problem solved by the users of each school
+     *
+     * @param  int $startTime
+     * @param  int $finishTime
+     * @param  int $offset
+     * @param  int $rowcount
+     * @return array{school_id: int, name: string, country_id: string, score: float}[]
+     */
+    public static function getRankByProblemsScore(
+        int $startDate,
+        int $finishDate,
+        int $offset,
+        int $rowcount
+    ): array {
+        $sql = '
+            SELECT
+                s.school_id,
+                s.name,
+                s.country_id,
+                SUM(ROUND(100 / LOG(2, p.accepted+1), 0)) AS score
+            FROM
+                Schools s
+            INNER JOIN
+                Submissions su ON su.school_id = s.school_id
+            INNER JOIN
+                Runs r ON r.run_id = su.current_run_id
+            INNER JOIN
+              Problems p ON p.problem_id = su.problem_id
```

Comment:
```suggestion
                Problems p ON p.problem_id = su.problem_id
```

así está alineado. Igual en L155 y L157.